presto-docs/src/main/sphinx/release/release-335.rst

```diff
@@ -0,0 +1,51 @@
+=========================
+Release 335 (XX Jun 2020)
+=========================
+
+General Changes
+---------------
+
+* Fix failure when :func:`reduce_agg` is used as a window function. (:issue:`3883`)
+* Fix incorrect cast from ``TIMESTAMP WITHOUT TIME ZONE`` to ``TIME`` type. (:issue:`3848`)
+* Fix incorrect query results when converting very large ``TIMESTAMP`` values into
+  ``TIMESTAMP WITHOUT TIME ZONE``, or when parsing very large
+  ``TIMESTAMP WITHOUT TIME ZONE`` values. (:issue:`3956`)
+* Improve optimized local scheduling with regard to non-uniform data distribution. (:issue:`3922`)
+* Add support for variable-precision ``TIMESTAMP WITHOUT TIME ZONE`` type. (:issue:`3783`)
+* Return ``VARCHAR`` type when :func:`substr` argument is ``CHAR`` type. (:issue:`3599`)
+* Add a variant of :func:`substring` that takes a ``CHAR`` argument. (:issue:`3949`)
+
+Security Changes
+----------------
+
+* Add  ``information_schema.role_authorization_descriptors`` table that returns information about the roles granted to principals. (:issue:`3535`)
+* Add schema access rules to file-based system access control. (:issue:`3766`)
+
+Hive Connector Changes
+----------------------
+
+* Fix a bug that caused missing column statistics after ``INSERT`` into a new table. (:issue:`2469`)
+* Add support for ``information_schema.role_authorization_descriptors`` table when ``hive.security=sql-standard``.  (:issue:`3535`)
+* Allow non-lowercase column names in ``system.sync_partition_metadata`` procedure. This can be enabled
+  by passing ``case_sensitive=false`` when invoking the procedure. (:issue:`3431`)
+* Support caching with secured coordinator. (:issue:`3874`)
+* Prevent caching from turning off in case of intermittent network failures. (:issue:`3874`)
+* Make caching and HDFS impersonation mutually exclusive. (:issue:`3913`)
+* Add ``hive.cache.ttl`` and ``hive.cache.disk-usage-percentage`` cache properties. (:issue:`3840`)
+* Schedule work on nodes with cached data. (:issue:`3922`)
```

Comment:
From end-user perspective this could be
```
Improve query performance when caching is enabled by [...]
```
?

Reply:
that would be good